src/BuildExecutable.jl

```diff
@@ -107,8 +111,9 @@ function build_executable(exename, script_file, targetdir=nothing, cpu_target="n
     emit_cmain(cfile, exename, targetdir != nothing)
     emit_userimgjl(userimgjl, script_file)
 
-    println("running: $(julia) $(build_sysimg) $(sys.buildfile) $(cpu_target) $(userimgjl) --force")
-    run(`$(julia) $(build_sysimg) $(sys.buildfile) $(cpu_target) $(userimgjl) --force`)
+    empty_cmd_str = ``
```

Comment:
I see. Could you move it to line 116 instead as it is not used any where else. So the end of line 116 would be ```$(debug ? "--debug" : `` )```